scripts-dev/complement.sh

```diff
@@ -231,6 +231,13 @@ else
   test_tags="$test_tags,faster_joins,msc2716"
 fi
 
+if [[ -n "$ASYNCIO_REACTOR" ]]; then
+  # Enable the Twisted asyncio reactor
+  export PASS_SYNAPSE_COMPLEMENT_USE_ASYNCIO_REACTOR=true
+else
+  export PASS_SYNAPSE_COMPLEMENT_USE_ASYNCIO_REACTOR=
```

Comment:
would probably just cut out this `else` clause

Reply:
Fair, will do.